pkg/services/folder/model.go

```diff
@@ -10,6 +10,7 @@ var ErrMaximumDepthReached = errutil.NewBase(errutil.StatusBadRequest, "folder.m
 var ErrBadRequest = errutil.NewBase(errutil.StatusBadRequest, "folder.bad-request")
 var ErrDatabaseError = errutil.NewBase(errutil.StatusInternal, "folder.database-error")
 var ErrInternal = errutil.NewBase(errutil.StatusInternal, "folder.internal")
+var ErrFolderTooDeep = errutil.NewBase(errutil.StatusInternal, "folder.tooDeep")
```

Comment:
These are inconsistent, but I think I'm the one who was wrong. What do you think - should this be `folder.too-deep`, or should we make the other errors  camel case?